lms/djangoapps/philu_overrides/tests/test_context_processor.py

```diff
@@ -0,0 +1,590 @@
+"""
+Tests for context processors in philu_overrides
+"""
+
+import mock
+import pytest
+from django.conf import settings
+from django.test import RequestFactory
+from django.urls import reverse
+
+from lms.djangoapps.philu_overrides.constants import (
+    ACTIVATION_ALERT_TYPE,
+    ACTIVATION_ERROR_MSG_FORMAT,
+    ORG_DETAILS_UPDATE_ALERT,
+    ORG_OEF_UPDATE_ALERT
+)
+from lms.djangoapps.philu_overrides.context_processor import (
+    add_nodebb_endpoint,
+    get_cdn_link,
+    get_global_alert_messages
+)
+from student.tests.factories import AnonymousUserFactory, UserFactory
+
+
+@pytest.fixture()
+def request_object(db):  # pylint: disable=unused-argument
+    request = RequestFactory().get('/dummy_url')
+    request.user = UserFactory.create()
```

Comment:
You can also use [build in fixtures ](https://pytest-django.readthedocs.io/en/latest/helpers.html)

```
@pytest.fixture()
def request_object(db, rf):
    request = rf.get('/dummy_url')
    ...
```

No need for RequestFactory  import